pages/linux/lvextend.md

```diff
@@ -1,6 +1,8 @@
 # lvextend
 
 > Increase the size of a logical volume.
+> One of the Logical Volume Manager (LVM) tools.
+> More information: <https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/7/html/logical_volume_manager_administration>.
```

Comment:
Isn't there a more neutral page we could include here? LVM is found on many distributions, not just RHEL.

Reply:
I've looked for other sources but didn't find any of this quality. FWIW, I believe Red Hat is either the originator or the sponsor of the project.

Reply:
This page isn't a comprehensive guide but would be more suited to someone just starting out and wanting to learn more: https://www.digitalocean.com/community/tutorials/an-introduction-to-lvm-concepts-terminology-and-operations.

Reply:
I see - that's quite surprising! In that case, let's keep this link as-is.

Reply:
Should I go ahead and make similar changes to the other lvm tool tldr pages?

Reply:
Good idea!  Go right ahead.  I'd open one big PR with a separate commit per page.

Reply:
Will do. Thanks for the guidance.